src/utils.rs

```diff
@@ -530,21 +532,21 @@ impl Preferences {
 		}
 		Self {
 			available_themes: themes,
-			theme: setting(&req, "theme"),
-			front_page: setting(&req, "front_page"),
-			layout: setting(&req, "layout"),
-			wide: setting(&req, "wide"),
-			show_nsfw: setting(&req, "show_nsfw"),
-			blur_nsfw: setting(&req, "blur_nsfw"),
-			use_hls: setting(&req, "use_hls"),
-			hide_hls_notification: setting(&req, "hide_hls_notification"),
-			autoplay_videos: setting(&req, "autoplay_videos"),
-			disable_visit_reddit_confirmation: setting(&req, "disable_visit_reddit_confirmation"),
-			comment_sort: setting(&req, "comment_sort"),
-			post_sort: setting(&req, "post_sort"),
-			subscriptions: setting(&req, "subscriptions").split('+').map(String::from).filter(|s| !s.is_empty()).collect(),
-			filters: setting(&req, "filters").split('+').map(String::from).filter(|s| !s.is_empty()).collect(),
-			hide_awards: setting(&req, "hide_awards"),
+			theme: setting(req, "theme"),
```

Comment:
Vis-a-vis these `&req` to `req` changes, see my comment on src/server.rs:382.

Reply:
TBH I think it does make a lot of sense to have this change together with the fix as the fix directly modifies the code in question.